doc/release-notes.md

```diff
@@ -1,6 +1,126 @@
+  - This particular release added the following datasets:
+    - 11 RNA-Seq and WXS PBTA samples (plus 2 WXS PBTA results from previous release)
+    - 1689 DGD targeted sequencing tumor samples
+    - methylation samples
+      - methylation samples: 
+        - 5 PBTA samples
+        - 12 TARGET samples
+      - methylation sample type not reported
+        - 3 PBTA 
+      - methylation tumor samples
+        - 1143 PBTA samples
+        - 751 TARGET samples
+      - RNA isoform merged TPM samples
+  - Update TCGA RNA-Seq data with GENCODE v36 GDC
+  - All additional changes as well as details about the above changes are as followed.
+
+- Add base histologies file to data release, see [ticket 333](https://github.com/PediatricOpenTargets/ticket-tracker/issues/333)
+- Update histologies file - key changes noted here (other tickets can be found in OpenPedCan repo)
+    - Change cancer_group of `Metastatic secondary tumors;Neuroblastoma` to `Neuroblastoma`
+            - See discussion in [ticket 232](https://github.com/PediatricOpenTargets/ticket-tracker/issues/232)
+            - [Ticket 234](https://github.com/PediatricOpenTargets/ticket-tracker/issues/234)
+    - Update `primary_site` for some GMKF and TARGET NBL samples to resolve lumped together `primary_site` issues
+            - [Ticket 257](https://github.com/PediatricOpenTargets/ticket-tracker/issues/257)
+    - Update histologies file to use `pathology_diagnosis` (when available) or `cancer_group` (when path dx not available) to fill in `harmonized_diagnosis` 
+            - [Ticket 259](https://github.com/PediatricOpenTargets/ticket-tracker/issues/259)
+     - DGD histologies cleanup: [ticket 325](https://github.com/PediatricOpenTargets/ticket-tracker/issues/325)
+     - Short histology for PBTA cohort does not match: [ticket 309](https://github.com/PediatricOpenTargets/ticket-tracker/issues/309)
+     - Add extent of tumor regression: [ticket 298](https://github.com/PediatricOpenTargets/ticket-tracker/issues/298)
+
+- Update EFO MONDO file 
+    - Update EFO MONDO map file to contain new cancer group in v10 histologies file: [ticket 237](https://github.com/PediatricOpenTargets/ticket-tracker/issues/237)
+    - Add NCIT column to the EFO MONDO map file + add new AML terms: [ticket 261](https://github.com/PediatricOpenTargets/ticket-tracker/issues/261)
+- Update `ensg-hugo-pmtl-mapping.tsv` to remove NA line: [ticket 231](https://github.com/PediatricOpenTargets/ticket-tracker/issues/231)
+
+- Re-run independent samples module since we would have cancer group update for the histologies file and addition of samples: [ticket 304](https://github.com/PediatricOpenTargets/ticket-tracker/issues/304)
+- Add concatenated DGD MAF files to data release as `snv-dgd.maf.tsv.gz`: [ticket 248](https://github.com/PediatricOpenTargets/ticket-tracker/issues/248)
+- Add concatenated DGD fusion files as `fusion-dgd.tsv.gz`: [ticket 249](https://github.com/PediatricOpenTargets/ticket-tracker/issues/249)
```

Comment:
move this up to earlier when showing DGD sample N (can just add ticket links and not need to describe